### PR TITLE
Add task stats to user dashboard

### PR DESCRIPTION
**Pull request by danielfdsilva**
To review only after this https://github.com/hotosm/imagery-requests/pull/53 is merged.

Requires latest develop on the api https://github.com/hotosm/imagery-requests-api

### CODE REVIEW

File: app/assets/scripts/views/dashboard.js [hunk 1]
```diff
@@ -126,23 +140,41 @@ var Dashboard = React.createClass({
...
+    if (!fetched) {
+      return null;
+    }
+
+    return (
+      <div className='tasks-stats'>
+        <h2 className='tasks-stats__title'>Tasks</h2>
+        <ul className='stats-list'>
+          <li className='stats-list__item'><strong>{data.stats.activeTasks}</strong>active</li>
+          <li className='stats-list__item'><strong>{data.stats.completedTasks}</strong>completed</li>
```
Comment (nbumbarger) [inline app/assets/scripts/views/dashboard.js:155]: this returns undefined; it needs to reference `data.stats.completeTasks`

**Review by nbumbarger — CHANGES_REQUESTED** [review]
- As the surveyor, I changed the status of two requests from "completed" to "in progress". They're now labeled as "in progress," but the stats still show 0 active, 2 completed.
- I think that submitting a change to a task status should return the user to their previous view, or at least give some indication that the input was received.

File: app/assets/scripts/views/dashboard.js (hunk 1 above)
Comment (danielfdsilva) [reply app/assets/scripts/views/dashboard.js:155]: This was my bad. I made a change to ensure consistent naming but forgot to push :S
Here it is: https://github.com/hotosm/imagery-requests-api/pull/16

**Comment by danielfdsilva** [comment]
@nbumbarger 
1) Where are you seeing those statuses?
2) When you submit a task status change it immediately appears below the form. What do you mean by return to the previous view?

**Comment by nbumbarger** [comment]
@danielfdsilva

1. The stat mismatch is visible in the dashboard- closing a task, then reopening it, doesn't update the number of open/ closed tasks displayed at the top of the page.
2. This isn't a real issue, but I think there should be more visual feedback when adding comments. It's hard to notice the addition. Maybe an animation that slides the comments down to make room?

**Pull request merged**